src/docs/product/accounts/membership.mdx

```diff
@@ -9,87 +9,130 @@ description: "Learn about the different types of Sentry membership, roles, and a
-  only through invitation by a Manager, Owner, or Admin.
+Org Members can be Team Admins for specific teams, but they can't add new teams. Org Admins automatically become Team Admins when they join a team. Org Managers and Owners can perform Team Admin actions for all teams without needing to join.
 
-To restrict Team access, go to Organization Settings and flip the “Open Membership” toggle.
+### Permissions
+
+Team-level roles enable more granular access to a specific team, as described in the table below:
+
+| **Action**                                                                                          | **Team Contributor** | **Team Admin** |
+| --------------------------------------------------------------------------------------------------- | -------------------- | -------------- |
+| Can view and act on issues (such as assigning and resolving)                                        | ✓                    | ✓              |
+| Can join and leave teams (depending on open membership settings)                                    | ✓                    | ✓              |
+| Can add repositories                                                                                | ✓                    | ✓              |
+| Can invite new members to join (_Invite lands in pending state for Managers and Owners to approve_) | ✓                    | ✓              |
+| Can change project settings                                                                         |                      | ✓\*            |
+| Can assign Team Admin Role                                                                          |                      | ✓\*            |
+| Can add, edit, or remove Team Contributors                                                          |                      | ✓\*            |
+| Can add projects for teams                                                                          |                      | ✓\*            |
+| Can remove projects from teams                                                                      |                      | ✓\*            |
+| Can create alerts even if the org setting for "Let members create alerts" is disabled               |                      | ✓\*            |
+| Can create new projects for the organization                                                        |                      | ✓\*            |
+| Can remove an existing team from the organization                                                   |                      | ✓\*            |
+| Can remove existing projects from the organization                                                  |                      | ✓\*            |
+| Can create new teams for the organization                                                           |                      |                |
+| Can transfer existing projects to another organization                                              |                      |                |
```

Comment:
Rearranged the sequence so it goes from org-level actions, to team-level actions, to project-level actions.

```suggestion
| Can invite new members to join (_Invite lands in pending state for Managers and Owners to approve_) | ✓                    | ✓              |
| Can join and leave teams (depending on open membership settings)                                    | ✓                    | ✓              |
| Can view and act on issues (such as assigning and resolving)                                        | ✓                    | ✓              |

| Can add repositories                                                                                | ✓                    | ✓              |
| Can create new teams for the organization                                                           |                      |                |
| Can remove an existing team from the organization                                                   |                      | ✓\*            |
| Can assign Team Admin Role                                                                          |                      | ✓\*            |
| Can add, edit, or remove Team Contributors                                                          |                      | ✓\*            |
| Can create new projects for the organization                                                        |                      | ✓\*            |
| Can remove existing projects from the organization                                                  |                      | ✓\*            |
| Can add projects for teams                                                                          |                      | ✓\*            |
| Can remove projects from teams                                                                      |                      | ✓\*            |
| Can change project settings                                                                         |                      | ✓\*            |
| Can create alerts even if the org setting for "Let members create alerts" is disabled               |                      | ✓\*            |
```